File: athrill-account-client/projects/at-models/src/lib/authentication/authentication-response.model.ts
import {UserModel} from './../users/user.model';

export class AuthenticationResponseModel
{
    public Token : string;
    public TokenExpireTTL : Date;
    public IsResetPassword? : boolean;
    public UserRoleId : number;
    public User : UserModel;
    public Permissions : string[];
}

File: athrill-account-server/Libraries/Model/Authentication/AuthenticationResponseModel.cs
using AT.Common.Constants;
using AT.Model.Users;
using System;
using System.Collections.Generic;

namespace AT.Model.Authentication
{
    public class AuthenticationResponseModel
    {
        public string Token { get; set; }
        public DateTime TokenExpireTTL { get; set; }
        public bool? IsResetPassword { get; set; }
        public UserModel User { get; set; }
        public int UserRoleId { get; set; }
        public List<string> Permissions { get; set; }
    }
}
